README.md

```diff
@@ -233,13 +233,154 @@ mstorage.info("<media_id>", new CompletionHandler<MediaInfo>() {
 });
 ```
 
-### Get media metadata
+### Get all media metadata
 ```java
 mstorage.meta("<media_id>", new CompletionHandler<MediaMeta>() {
     @Override
     public void onCompleted(MediaMeta mediaMeta) {
         Map<String, String> exif = mediaMeta.getExif();
         Map<String, String> gpano = mediaMeta.getGpano();
+        Map<String, String> userMeta = mediaMeta.getUserMeta();
+        // Do something.
+    }
+
+    @Override
+    public void onThrowable(Throwable t) {
+        // Something wrong happened.
+    }
+});
+```
+
+### Get specific media metadata
+
+* Exif
+
+```java
+mstorage.exif("<media_id>", new CompletionHandler<Map<String, String>>() {
+    @Override
+    public void onCompleted(Map<String, String> exif) {
+        // Do something.
+    }
+
+    @Override
+    public void onThrowable(Throwable t) {
+        // Something wrong happened.
+    }
+});
+```
+
+* Google Photo Sphere XMP
+
+```java
+mstorage.gpano("<media_id>", new CompletionHandler<Map<String, String>>() {
+    @Override
+    public void onCompleted(Map<String, String> gpano) {
+        // Do something.
+    }
+
+    @Override
+    public void onThrowable(Throwable t) {
+        // Something wrong happened.
+    }
+});
+```
+
+* All user metadata
+
+```java
+mstorage.userMeta("<media_id>", new CompletionHandler<Map<String, String>>() {
+    @Override
+    public void onCompleted(Map<String, String> userMeta) {
+        // Do something.
+    }
+
+    @Override
+    public void onThrowable(Throwable t) {
+        // Something wrong happened.
+    }
+});
+```
+
+* Specific user metadata
+
+```java
+mstorage.userMeta("<media_id>", "<key>", new CompletionHandler<String>() {
+    @Override
+    public void onCompleted(String value) {
+        // Do something.
+    }
+
+    @Override
+    public void onThrowable(Throwable t) {
+        // Something wrong happened.
+    }
+});
+```
+
+### Attach user metadata to media
```

Comment:
Maybe it's more natural to change this to "Attach metadata" and add a line "You can define your original metadata as a 'user metadata'."